backend/FwHeadless/Program.cs

```diff
@@ -35,7 +37,16 @@
 
 app.UseHttpsRedirection();
 
-app.MapHealthChecks("/api/healthz");
+app.MapHealthChecks("/api/healthz", new HealthCheckOptions
+{
+    ResponseWriter = async (context, healthReport) =>
+    {
+        var version = typeof(Program).Assembly.GetCustomAttribute<AssemblyInformationalVersionAttribute>()?.InformationalVersion ?? "dev";
+        context.Response.Headers["lexbox-version"] = version;
```

Comment:
I'd rather just add the header to all requests like we do in lexbox api

Reply:
Done in commit f1034055.